build/azure-pipelines/web/build/Dockerfile

```diff
@@ -8,7 +8,14 @@ RUN apt-get update && apt-get upgrade -y
 
 RUN apt-get install -y libxkbfile-dev pkg-config libsecret-1-dev libxss1 dbus xvfb libgtk-3-0 \
 	libkrb5-dev git apt-transport-https ca-certificates curl gnupg-agent software-properties-common \
-	libnss3 libasound2 make gcc libx11-dev fakeroot rpm libgconf-2-4 libunwind8 g++ python
+	libnss3 libasound2 make gcc libx11-dev fakeroot rpm libgconf-2-4 libunwind8 g++ python wget
+
+# Set the Chrome repo.
+RUN wget -q -O - https://dl-ssl.google.com/linux/linux_signing_key.pub | apt-key add - \
+    && echo "deb http://dl.google.com/linux/chrome/deb/ stable main" >> /etc/apt/sources.list.d/google.list
+
+# Install Chrome.
+RUN apt-get update && apt-get -y install google-chrome-stable
```

Comment:
we run `apt-get update` on line 7.  Probably reordering commands would eliminate the need to call it twice?

Reply:
Running `apt-get install -y google-chrome-stable` without running `apt-get update` first resulted in an error. The error would say that the "google-chrome-stable" package couldn't be located, as seen here when running with curl:

![image](https://user-images.githubusercontent.com/87730006/134730973-aa136296-f3a5-4051-ad15-7990836f9a35.png)